test/import_test.js

```diff
@@ -31,35 +44,21 @@ describe.skip("/import", function () {
 		const name = translator.replace(/\.js$/, '');
 		const code = fs.readFileSync(path.join(translators, translator), 'utf-8');
 
-		let start = code.indexOf('/** BEGIN TEST CASES **/');
-		if (start === -1) continue;
-		let cases = null;
-
-		try {
-			start = code.indexOf('[', start);
-			if (start === -1) throw new Error('Could not find start of test cases');
+		const marker = code.indexOf('/** BEGIN TEST CASES **/');
+		if (marker === -1) continue;
 
-			const end = code.lastIndexOf(']') + 1;
-			if (end === -1) throw new Error('Could not find end of test cases');
+		const header = parseJSON(name, code.replace(/\n}\n[\s\S]*/, '}'), code);
+		if (!header) continue;
+		const start = code.indexOf('[', marker);
+		const end = code.lastIndexOf(']') + 1;
 
-			cases = code.substring(start, end);
-			let caseNo = 0;
-			for (const test of JSON.parse(cases)) {
-				caseNo += 1;
-				if (test.type === 'import') addTest(`${name} # ${caseNo}`, test.input, test.items);
-			}
+		const cases = parseJSON(name, code.substring(Math.max(start, marker), Math.max(end, start, marker)), code);
+		if (!cases) continue;
 
-		} catch (err) {
-			it(`should import ${name}`, function () {
-				const m = err.message.match(/(.*) at position ([0-9]+)$/);
-				if (m && typeof cases === 'string') {
-					const pos = parseInt(m[2]);
-					const line = code.substring(0, start + pos).split('\n').length;
-					const column = pos - cases.lastIndexOf('\n', pos);
-					throw new Error(`${m[1]} around line ${line}, column ${column}`);
-				}
-				throw err;
-			});
+		let caseNo = 0;
+		for (const test of cases) {
+			caseNo += 1;
+			if (test.type === 'import') addTest(`${name} # ${caseNo}`, header.translatorID, test.input, test.items);
```

Comment:
New line and braces (or `if (test.type != 'import') continue;`)

Reply:
Done, but the .eslintrc proposed for the translators allows this form.

Reply:
It does, but we try to keep to a line length of 100, and we mostly only use the single-line format for lines like `if (foo) continue;` where readability isn't a concern.

Reply:
Is it likely https://github.com/zotero/translators/pull/1895 is going to be merged soon? I want to get started on fixing the import tests but some of these will involve code changes, and I'd prefer to have the linter at hand for that.